clients/ui/bff/data/registered_model.go

```diff
@@ -39,3 +39,18 @@ func CreateRegisteredModel(client integrations.HTTPClientInterface, jsonData []b
 
 	return &model, nil
 }
+
+func GetRegisteredModel(client integrations.HTTPClientInterface, id string) (*openapi.RegisteredModel, error) {
+	responseData, err := client.GET(registerModelPath + "/" + id)
```

Comment:
Maybe it would be nice to edit our `GET()` so that you can just pass a list for the path and not have to worry about adding `/` in between everything. The `GET()` could piece together our URL for us.

Reply:
I'm not against this idea, but that's a reasonably sizeable refactor (in terms of code changes as opposed to effort) - you have to modify every call site to the GET func and IMO this kind of change belongs as a separate standalone PR.

Reply:
Now addressing your original issue, I'm not a massive fan of switching to taking a slice as it complicates the calling syntax a bit....

e.g. `responseData, err := client.GET([]string{registerModelPath, id})`

the slice literal isn't immediately intuitive to me, also it's rather annoying when you only have a single string.

Now a better solution might be to use a variadic `path ...string` param - this actually works really nicely - there's one gotcha here that varargs have to be the last param - so for the POST and other methods, we need to slip them to take other params (potentially optional) params first which isn't as nice.

WDYT?

Note: as an extra aside, if you're a big fan of syntactic sugar, go is 100% not your language, it's pretty much designed on the principal that syntactic sugar is bad :P